src/functions/sendCandidateResults/application/service/personalisation-provider.ts

```diff
@@ -70,4 +71,8 @@ export class PersonalisationProvider implements IPersonalisationProvider {
     }
     return '';
   }
+
+  private getApplicationRef (ref: ApplicationReference) {
+    return `${ref.applicationId }${ref.bookingSequence}${ref.checkDigit}`;
```

Comment:
Extra space here, don't suppose it breaks anything.

Reply:
Removed it nice spot